{"author":{"id":"1c0be4c7efe4cf70011d270a87e33999699eb0e9a5b12f84e0369f90dae3dacd"},"ops":[{"type":3,"timestamp":1638563719,"nonce":"oxG3r4hmmt4vTJG1AkaQ6vfNKzg=","metadata":{"github-id":"IC_kwDOCGKeh846wkft","github-url":"https://github.com/git-bug/git-bug/issues/723#issuecomment-985810925"},"message":"How long did the import run until you hit this error?","files":null}]}